examples/text-generation/utils.py

```diff
@@ -181,6 +181,32 @@ def get_torch_compiled_model(model):
     return model
 
 
+def setup_quantization(model, args):
+    if os.getenv("USE_INC", "1") != "0":
```

Comment:
Can this be made a command line option instead?

Reply:
We should plan for the behavior of the majority of future versions.
1.17.0 is going to be released soon and HQT is deprecated so ideally use inc is the default and USE_HQT should be the explicit setting:
If we flip the code to handle 
if os.getenv("USE_HQT", "0") == "1":
 #hqt
else:
  #inc
its easier to maintain going forward...otherwise everyone has to set USE_INC=1 after v1.17.0 release. But let me know if I misunderstood anything.